src/test/java/no/unit/nva/model/instancetypes/BookMonographTest.java

```diff
@@ -0,0 +1,145 @@
+package no.unit.nva.model.instancetypes;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.model.exceptions.InvalidPageTypeException;
+import no.unit.nva.model.pages.MonographPages;
+import no.unit.nva.model.pages.Pages;
+import no.unit.nva.model.pages.Range;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class BookMonographTest {
+
+    private final ObjectMapper objectMapper = JsonUtils.objectMapper;
+
+    @DisplayName("BookMonograph exists")
+    @Test
+    void bookMonographExists() {
+        new BookMonograph();
+    }
+
+    @DisplayName("BookMonograph: ObjectMapper correctly deserializes object")
+    @Test
+    void objectMapperReturnsBookMonographWhenInputIsValid() throws JsonProcessingException {
+        String expectedIntroductionBegin = "i";
+        String expectedIntroductionEnd = "xxviii";
+        String expectedPages = "398";
```

Comment:
It is a bit confusing because when I see all three lines (33-35) together.  Somehow I do in my head  the calculation (xxviii -i) and I expect it to be 398. Why is it important for the introduction pages to be in this test? Can you make another test where you say for example,
`getPagesDoesNotIncludeIntroductoryPages()` or sth like that?